extensions/caffeine/deployment/src/main/java/io/quarkus/caffeine/deployment/devui/CaffeineDevUIProcessor.java

```diff
@@ -22,27 +14,12 @@ public class CaffeineDevUIProcessor {
     void createCard(BuildProducer<CardPageBuildItem> cardPageBuildItemBuildProducer) {
         final CardPageBuildItem card = new CardPageBuildItem();
 
-        final PageBuilder versionPage = Page.externalPageBuilder("Version")
-                .icon("font-awesome-solid:mug-hot")
-                .url("https://github.com/ben-manes/caffeine")
-                .doNotEmbed()
-                .staticLabel(getManifest(Caffeine.class).getMainAttributes().getValue("Bundle-Version"));
-        card.addPage(versionPage);
+        card.addLibraryVersion("com.github.ben-manes.caffeine", "caffeine", "Caffeine",
+                "https://github.com/ben-manes/caffeine");
 
-        card.setCustomCard("qwc-caffeine-card.js");
+        card.setLogo("logo_dark.png", "logo_light.png");
```

Comment:
could we also default to the icon-url that is in `quarkus-extension.yaml` like in Quarkus JGit it has this

```yaml
icon-url: "https://raw.githubusercontent.com/quarkiverse/quarkus-jgit/main/docs/modules/ROOT/assets/images/jgit.svg"
```

Reply:
There is no dark / light variation in that one , but let me see what I can do